### PR TITLE
Feature/visualdl mnist

### PR DESCRIPTION
**Pull request by dzhwinter**
check for debug

### CODE REVIEW

File: fluid/mnist.py [hunk 1]
```diff
@@ -161,6 +169,11 @@ def run_benchmark(model, args):
             end = time.time()
             loss = np.array(outs[0])
             acc = np.array(outs[1])
+
+            loss_scalar.add_record(step, loss)
```
Comment (Superjomn) [inline fluid/mnist.py]: loss 必须是一个 scalar，就是个数字，而非 numpy 或 list

pybind 直接会类型错误

Comment (dzhwinter) [reply fluid/mnist.py]: Please throw an exception instead of terminating.
